app/controllers/cases_controller.rb

```diff
@@ -363,6 +366,9 @@ def update_closure
 
     @case.prepare_for_close
     close_params = process_closure_params(@case.type_abbreviation)
+    if @case.ico?
+      @s3_direct_post = S3Uploader.s3_direct_post_for_case(@case, 'responses')
```

Comment:
could you change `responses` to `ico_decisions`

Reply:
yep